calicoctl/resourcemgr/resourcemgr.go

```diff
@@ -165,7 +165,7 @@ func (rh resourceHelper) Apply(ctx context.Context, client client.Interface, res
 	// Fall back to an Update if the resource already exists, or the datastore does not support
 	// create operations for that resource.
 	switch err.(type) {
-	case cerrors.ErrorResourceAlreadyExists, cerrors.ErrorOperationNotSupported:
+	case cerrors.ErrorResourceAlreadyExists:
```

Comment:
Comment above is now incorrect.

Reply:
The comment and code seem to be deliberate, so I'm wondering if there are some resources for which Create is not supported, but Apply is - I'm thinking  maybe the global config or cluster info types.  Could you do a quick check in the code?

Reply:
I believe it is only k8s network policy and clusterinfo which can return NotSupported, and its consistent across all CRUD operations for them. So no, afaict, there are no resources where Create is not supported but Apply is.